src/Doctrine/Instantiator/Instantiator.php

```diff
@@ -20,44 +22,40 @@
 use function strlen;
 use function unserialize;
 
-use const PHP_VERSION_ID;
-
 final class Instantiator implements InstantiatorInterface
 {
     /**
      * Markers used internally by PHP to define whether {@see \unserialize} should invoke
      * the method {@see \Serializable::unserialize()} when dealing with classes implementing
      * the {@see \Serializable} interface.
      */
-    public const SERIALIZATION_FORMAT_USE_UNSERIALIZER   = 'C';
-    public const SERIALIZATION_FORMAT_AVOID_UNSERIALIZER = 'O';
+    private const SERIALIZATION_FORMAT_USE_UNSERIALIZER   = 'C';
+    private const SERIALIZATION_FORMAT_AVOID_UNSERIALIZER = 'O';
```

Comment:
Those constants are never used publicly and I believe they should not be part of the public interface. Do we need to flag them as deprecated on 1.x or is it enough to just switch them to private in 2.0?

Reply:
Let's flag them as deprecated out of consistency with https://github.com/doctrine/orm/pull/10342?

Reply:
Done in #96